common/personalities/hfe_personality.txt

```diff
@@ -242,169 +246,281 @@ hfe_personality_ahet_primitive_0 = {
 }
 
 hfe_personality_varelse_primitive_0_0 = {
-	aggressiveness = 0
-	trade_willingness = 0
-	bravery = 0
-	combat_bravery = 0
-	military_spending = 0
-	colony_spending = 0
+	aggressiveness = 0.6
+	trade_willingness = 1.1
+	bravery = 0.5
+	combat_bravery = 1
+	military_spending = 3
+	colony_spending = 1.1
 	federation_acceptance = 0
-	nap_acceptance = 0
-	defensive_pact_acceptance = 0
-	migration_pact_acceptance = 0
-	weapon_preferences = weapon_type_energy
-	armor_ratio = 0
-	shields_ratio = 0
-	threat_modifier = 0
-	threat_others_modifier = 0
-	friction_modifier = 0
-	claims_modifier = 0
+	nap_acceptance = 50
+	defensive_pact_acceptance = 50
+	migration_pact_acceptance = -10
+	research_agreement_acceptance = -1000
+	weapon_preferences = weapon_type_explosive
+	hull_ratio = 0.25
+	armor_ratio = 0.5
+	shields_ratio = 0.25
+	threat_modifier = 1
+	threat_others_modifier = 1.2
+	friction_modifier = 0.2
+	claims_modifier = 10
 	behaviour = {
 		conqueror = no
-		subjugator = no
-		liberator = no
-		opportunist = no
+		subjugator = yes
+		liberator = yes
+		opportunist = yes
 		slaver = no
-		uplifter = no
+		uplifter = yes
 		purger = no
 		displacer = no
 		infiltrator = no
-		dominator = no
-		robot_exploiter = no
+		dominator = yes
+		robot_exploiter = yes
 		robot_liberator = no
-		propagator = no
+		propagator = yes
 		multispecies = no
-		crisis_fighter = no
+		crisis_fighter = yes
+		wants_tribute = yes
 	}
 	allow = {
+#		has_country_flag = hfe_varelse
 		has_country_flag = hfe_varelse_primitive_0
-		is_country_type = primitive
```

Comment:
this intended only for primitive

Reply:
**_NOT ANYMORE IT ISN'T!_**

I worked my butt off making the Cult what it is now. There are three Varelse Personalities. One is if you topple the Cult, one is for the Cult, and one is for the Varelse Hegemony that forms when the Fleet Trapped In Time settles down.

This was my most important contribution to the Mod in terms of flavor and story.